charts/enterprise/prometheus/templates/prometheus/_prometheus.tpl

```diff
@@ -126,8 +126,9 @@ spec:
   {{- if .Values.prometheus.remoteRead }}
   remoteRead: {{- include "tc.v1.common.tplvalues.render" (dict "value" .Values.prometheus.remoteRead "context" $) | nindent 4 }}
   {{- end }}
-  {{- if .Values.prometheus.remoteWrite }}
-  remoteWrite: {{- include "tc.v1.common.tplvalues.render" (dict "value" .Values.prometheus.remoteWrite "context" $) | nindent 4 }}
+  {{- with .Values.prometheus.remoteWrite }}
+  remoteWrite:
+    {{- toYaml . | nindent 4 }}
```

Comment:
```suggestion
    {{- tpl (toYaml .) $ | nindent 4 }}
```
Probably the function was having this aswell

Reply:
But probably we should reimplement this in common as helper. I’ll look into it later

Reply:
> But probably we should reimplement this in common as helper. I’ll look into it later

Found the old template, it wasn't much and its resulting more or less in the same line count anyway without any benefit of abstraction.

Reply:
Cool. If this works I can go through and clean up the rest of these in a followup PR.